### ISSUE TITLE
Unable to scan Shell scripts with current documentation

### ISSUE THREAD

**Issue by koosappa** [lequal/sonar-icode-cnes-plugin#40]
### Expected behavior

Scanned result following readme instructions.

### Actual behavior

README.md does not contains enough information to run an analysis.
```
INFO: Source paths: Desktop/templar_Code/Test/home/templar/legacy/test.ksh
INFO: Source encoding: windows-1252, default locale: en_US
INFO: Load server rules
INFO: Load server rules (done) | time=530ms
INFO: Index files
INFO: 1 file indexed
INFO: Quality profile for shell: Sonar way
INFO: Sensor fr.cnes.sonar.plugins.icode.check.ICodeSensor [icode]
**WARN: Results file result.res has not been found and wont be processed.**
INFO: Sensor fr.cnes.sonar.plugins.icode.check.ICodeSensor [icode] (done) | time
=0ms
INFO: Sensor SonarJavaXmlFileSensor [java]
INFO: Sensor SonarJavaXmlFileSensor [java] (done) | time=0ms
INFO: Sensor Zero Coverage Sensor
INFO: Sensor Zero Coverage Sensor (done) | time=0ms
INFO: No SCM system was detected. You can use the 'sonar.scm.provider' property
to explicitly specify it.
INFO: Calculating CPD for 0 files
INFO: CPD calculation finished
INFO: Analysis report generated in 125ms, dir size=35 KB
```

### Detection version
**1.1.0**

**Comment by koosappa** [comment]
After run the report getting like Empty project in sonar Dash board.

**Comment by begarco** [comment]
Is it possible to have further information like your command line and the content of sonar-project.properties?

**Comment by koosappa** [comment]
Command :  sonar-scanner

After that :+1: 


INFO: SonarQube server 7.3.0
INFO: Default locale: "en_US", source code encoding: "UTF-8"
INFO: Publish mode
INFO: Load global settings
INFO: Load global settings (done) | time=141ms
INFO: Server id: BF41A1F2-AWVcWatnjLJJlvH75jSB
INFO: User cache: C:\Users\ADMIN\.sonar\cache
INFO: Load/download plugins
INFO: Load plugins index
INFO: Load plugins index (done) | time=63ms
INFO: Load/download plugins (done) | time=125ms
INFO: Loaded core extensions:
INFO: Process project properties
INFO: Load project repositories
INFO: Load project repositories (done) | time=250ms
INFO: Load quality profiles
INFO: Load quality profiles (done) | time=125ms
INFO: Load active rules
INFO: Load active rules (done) | time=2481ms
INFO: Load metrics repository
INFO: Load metrics repository (done) | time=47ms
WARN: SCM provider autodetection failed. No SCM provider claims to support this
project. Please use sonar.scm.provider to define SCM of your project.
INFO: Project key: Demo
INFO: Project base dir: C:\Users\ADMIN\Desktop\templar_Code\production_002\h
ome\templar\legacy
INFO: -------------  Scan demo_project
INFO: Base dir: C:\Users\ADMIN\Desktop\templar_Code\production_002\home\temp
lar\legacy
INFO: Working dir: C:\Users\ADMIN\Desktop\templar_Code\production_002\home\t
emplar\legacy\.scannerwork
INFO: Source paths: .
INFO: Source encoding: UTF-8, default locale: en_US
INFO: Load server rules
INFO: Load server rules (done) | time=749ms
INFO: Index files
INFO: 803 files indexed
INFO: Quality profile for shell: Sonar way
INFO: Sensor fr.cnes.sonar.plugins.icode.check.ICodeSensor [icode]
WARN: Results file C:/Users/ADMIN/Desktop/Dashboard+d3 has not been found an
d wont be processed.
INFO: Sensor fr.cnes.sonar.plugins.icode.check.ICodeSensor [icode] (done) | time
=0ms


Sonar.propertity file :+1: 

sonar.projectKey=ShellScan
sonar.projectName=ShellScan
sonar.projectVersion= 1.0
# Comma-separated paths to directories with sources (required)
sonar.sources=.
# Tells SonarQube where the i-Code CNES analysis reports are
sonar.icode.reports.path=C:/Users/ADMIN/Desktop/Dashboard+d3
# Encoding of the source files
sonar.sourceEncoding=UTF-8

**Comment by begarco** [comment]
You have to provide the path to the file in *sonar.icode.reports.path=C:/Users/ADMIN/Desktop/Dashboard+d3* including filename

**Comment by koosappa** [comment]
I have just provided sonar.icode.reports.path as desktop/folder..  i have tried without path also same error...

**Comment by begarco** [comment]
Does this file exists: C:/Users/ADMIN/Desktop/Dashboard+d3 ?

**Comment by koosappa** [comment]
folder exist C:/Users/ADMIN/Desktop/Dashboard+d3 not file . I need to create new res file ?

**Comment by koosappa** [comment]
Otherwise please provide me sonar.property data from your end.

**Comment by begarco** [comment]
Ok so you have to launch manually i-code CNES before running sonar-scanner.

You can do it with custom i-Code command line or by using autolauch integrated in the plugin. In both case you need i-Code to be installed on your system.

I will update the README of the project, there aren't enough information.
